apm-agent-core/src/main/java/co/elastic/apm/agent/bci/ElasticApmAgent.java

```diff
@@ -257,6 +259,7 @@ private static synchronized void initInstrumentation(final ElasticApmTracer trac
         Runtime.getRuntime().addShutdownHook(new Thread(ThreadUtils.addElasticApmThreadPrefix("init-instrumentation-shutdown-hook")) {
             @Override
             public void run() {
+                logger.info("Used instrumentation groups: {}", InstrumentationUsageUtil.getUsedInstrumentationGroups());
```

Comment:
Please move this part to `MatcherTimerLifecycleListener#stop` to combine these two related functionalities

Reply:
Done